### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 16

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#142] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-02 185327](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5e5827f3-549b-4834-b8f2-4dfcbee3250d)

---
![Screenshot 2023-10-28 110646](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6fe550a6-4daa-4bcf-b9a7-00a408291010)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/0763a4eb-7447-4abf-a56a-7b4b1651b18e)
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/27e0e852-4382-4d43-8cf0-6c73cc333c1b)

**Comment by cornflexxx** [comment, marked as answer]
![e16f4](https://github.com/sapienzastudentsnetwork/algebra2324/assets/121803863/6888430c-602c-48a2-850e-45626cfd7bc7)
Soluzione dalle dispense(Campanella)
